include/database_functions.php

```diff
@@ -205,15 +151,58 @@ function insertClass($type, $link, $classDate, $studentId){
 ");
 }
 
+//If there is a better way to handle this please let me know. So, I have a credits table (Every row in that table in actually a purchase the student made).If
```

Comment:
After we discussed this during Jog 3, I talked to Yuni, and we discussed that he would allow students to book a class even if they did not pay for it unless it is a new student. But if a student has taken classes with him during a certain time, they could book a class and pay for it afterward. I will have to add the ability to block/allow booking classes without credits. So, I thought that if we do so, it could be good to leave the refund table, so we know that negative entries in the credits table is a booked class that was not paid yet. What do you think about that?